tests/acceptance/test_build.py

```diff
@@ -61,21 +61,21 @@ def extract_partition(img, number):
 
 
 class TestBuild:
-    @pytest.mark.min_mender_version("1.0.0")
+    @pytest.mark.min_mender_version("2.5.0")
```

Comment:
Since this is a meta-mender-only change, I don't think the version here actually matters.

Reply:
Good point, I will remove.